packages/protocol/test/compatibility/ast-code.ts

```diff
@@ -1,139 +0,0 @@
-import { reportASTIncompatibilities } from '@celo/protocol/lib/compatibility/ast-code'
-import {
-  Change,
-  DeployedBytecodeChange,
-  MethodAddedChange,
-  MethodMutabilityChange,
-  MethodRemovedChange,
-  MethodReturnChange,
-  MethodVisibilityChange,
-  NewContractChange,
-} from '@celo/protocol/lib/compatibility/change'
-import { getTestArtifacts } from '@celo/protocol/test/compatibility/common'
-import { assert } from 'chai'
-
-const testCases = {
-  original: getTestArtifacts('original'),
-  original_copy: getTestArtifacts('original_copy'),
-  inserted_constant: getTestArtifacts('inserted_constant'),
-  added_methods_and_contracts: getTestArtifacts('added_methods_and_contracts'),
-  metadata_changed: getTestArtifacts('metadata_changed'),
-  big_original: getTestArtifacts('big_original'),
-  big_original_modified: getTestArtifacts('big_original_modified'),
-}
-
-// @ts-ignore
-const comp = (c1: Change, c2: Change): number => {
-  const v1 = JSON.stringify(c1)
-  const v2 = JSON.stringify(c2)
-  if (v1 === v2) {
-    return 0
-  }
-  return v1 < v2 ? 1 : -1
-}
-
-describe('#reportASTIncompatibilities()', () => {
```

Comment:
I feel like these test are important. What was the reason for deleting? Did the tests become irrelevant?

Reply:
yeah this was a mistake to remove it